gigasecond/gigasecond_test.spec.js

```diff
@@ -0,0 +1,23 @@
+import Gigasecond from './gigasecond';
+
+describe('Gigasecond', () => {
+
+  it('tells the gigasecond birth of a living person', () => {
+    let gs = new Gigasecond(new Date(1977, 5, 13));
+    let expectedDate = new Date(2009, 1, 19);
+    expect(gs.date()).toEqual(expectedDate);
+  });
+
+  xit('tells gigasecond birth of a future person', () => {
```

Comment:
Agreed, both should be timestamps. The only thing that matters is _not_ the day. It's the second. But a date assumes midnight.